app/javascript/components/pages/ApartmentEdit.js

```diff
@@ -1,9 +1,141 @@
 import React, { Component } from 'react'
 
+// form is added to apartment edit page and inputs are setting state on component
+
+
 export default class ApartmentEdit extends Component {
-  render() {
-    return (
-      <h3>ApartmentEdit</h3>
-    )
+    constructor(props) {
+        super(props)
+        this.state = {
+            form: {
+              street: '',
+              city: '',
+              state: '',
+              manager: '',
+              email: '',
+              price: '',
+              bedrooms: 0,
+              bathrooms: 0,
+              pets: '',
+            },
+          success: false,
```

Comment:
This looks good here, are you using this for a redirect?  If so, I don't see it.

Reply:
Changed it from success to submitted.